diff -up xdm-1.1.6/config/Xresources.cpp.bug470348 xdm-1.1.6/config/Xresources.cpp
--- xdm-1.1.6/config/Xresources.cpp.bug470348	2007-06-16 04:25:22.000000000 +0200
+++ xdm-1.1.6/config/Xresources.cpp	2008-11-07 22:03:29.000000000 +0100
@@ -36,6 +36,7 @@ xlogin*greetFace:	Serif-24:bold:italic
 xlogin*face: 		Helvetica-18
 xlogin*promptFace: 	Helvetica-18:bold
 xlogin*failFace: 	Helvetica-18:bold
+XHASHendif
 
 XHASHifdef COLOR
 xlogin*borderWidth: 1
